src/app/shared/issue-pr-card/issue-pr-card.component.html

```diff
@@ -1,11 +1,12 @@
 <mat-card class="card" [ngClass]="getIssueOpenOrCloseColorCSSClass()">
-  <a class="no-underline link-grey-dark" (click)="viewIssueInBrowser($event)">
+  <a class="no-underline link-grey-dark">
```

Comment:
This is a good PR that accurately addresses the issue!

Perhaps you could consider moving the style class to the `IssuePrCardHeader` as well to avoid an ambiguous situation such as the one below, where hovering over the milestone highlights the issue as well:
<img width="199" alt="Screenshot 2024-07-07 at 3 32 48 PM" src="https://github.com/CATcher-org/WATcher/assets/114080910/aa0db077-c7ff-47e9-a90c-cd5575cec3b7">

Reply:
Thank you for the review, I have updated the PR according to the suggestion so only hovering over the header component will display the issue/pr updated date.

Hovering over header:
![image](https://github.com/CATcher-org/WATcher/assets/108446221/c35fd830-49cd-4f06-8d12-e73ebe0c4ec5)

Hovering over milestone:
![image](https://github.com/CATcher-org/WATcher/assets/108446221/5babb90f-ed5a-4e85-8545-7db76dc6d2c9)